myria/schema.py

```diff
@@ -0,0 +1,9 @@
+class MyriaSchema(object):
+  def __init__(self, json):
+    self.json = json
+    self.names = json['columnNames']
+    self.types = json['columnTypes']
+
+  def toJson(self):
+    return {'columnNames': map(lambda s: s.encode('ascii'), self.names), 
```

Comment:
Weird, I only forced an ascii encoding because of an error when submitting plans to the backend, but now I'm unable to reproduce.